packages/common/src/store/purchase-content/slice.ts

```diff
@@ -15,6 +16,7 @@ type OnSuccess = {
 }
 
 type PurchaseContentState = {
+  page: PurchaseContentPage
```

Comment:
I think this is fine, but I also think it could just be state local to the component to track which page we're on

Reply:
but then would need plumbing.. also now we're gonna do the same thing in mobile where we just use 1 drawer for everything. will push update tmr.